homeassistant/components/history.py

```diff
@@ -197,14 +206,13 @@ def states_to_json(hass, states, start_time, entity_id, filters=None):
     """
     result = defaultdict(list)
 
-    entity_ids = [entity_id] if entity_id is not None else None
-
     # Get the states at the start time
     timer_start = time.perf_counter()
-    for state in get_states(hass, start_time, entity_ids, filters=filters):
-        state.last_changed = start_time
-        state.last_updated = start_time
-        result[state.entity_id].append(state)
+    if get_initial_state:
```

Comment:
I think that we should rename this to something that better explains what it does. What about `include_start_time_state` ?

Reply:
Done